packages/grid/_modules_/grid/components/menu/columnMenu/GridColumnMenu.tsx

```diff
@@ -1,29 +1,40 @@
-import MenuList from '@material-ui/core/MenuList';
 import * as React from 'react';
+import MenuList from '@material-ui/core/MenuList';
 import { GridColDef } from '../../../models/colDef/gridColDef';
 import { GridColumnsMenuItem } from './GridColumnsMenuItem';
 import { GridFilterMenuItem } from './GridFilterMenuItem';
 import { HideGridColMenuItem } from './HideGridColMenuItem';
 import { SortGridMenuItems } from './SortGridMenuItems';
+import { useId } from '../../../utils/material-ui-utils';
 
 export interface GridColumnMenuProps {
   hideMenu: () => void;
   currentColumn: GridColDef;
+  open: boolean;
 }
 
 export function GridColumnMenu(props: GridColumnMenuProps) {
-  const { hideMenu, currentColumn } = props;
+  const { hideMenu, currentColumn, open } = props;
+  const columnMenuButtonId = useId('column-menu-button');
+  const columnMenuId = useId('column-menu');
   const handleListKeyDown = React.useCallback(
     (event: React.KeyboardEvent) => {
-      if (event.key === 'Tab') {
+      if (event.key === 'Tab' || event.key === 'Escape') {
         event.preventDefault();
```

Comment:
I don't think prevent default should be called for <kbd>Escape</kbd>. There are no native action to prevent. We don't prevent in the main repo